README.md

```diff
@@ -10,6 +10,15 @@ This repository contains the published and work in progress artifacts, docs, sch
     - **api_docs** - contains component level swagger
   - **technical_masterplan_assets** - contains images used in the `technical_masterplan.d` document
 
+## Install
+-   To record important architecture decision, we use  [adr-tools](https://github.com/npryce/adr-tools "https://github.com/npryce/adr-tools"). To install, type the following in the terminal command. 
```

Comment:
To be clear, I do love this idea, but I do have concerns using this would be good for everyone who needs to write ADRs; the technical barriers might not be inclusive enough. That is, getting adoption for this might be hard. Wondering instead if we can have PRs to the Master Plan or other top artifacts be the decision record instead. Thoughts?

Reply:
The only concern I have is the options of what we are looking at will not be clear in the PRs. I will test this in VRS and see if I have success or fail in people's adoption.